README.md

```diff
@@ -1,60 +1,36 @@
 # Datadog Integration Template
 
```

Comment:
This repository is *meant* to help create *tiles*

Reply:
Oh god I forgot to spell check before committing, haha. _embarrassed.gif_